site/docs/v1/tech/core-concepts/tenants.adoc

```diff
@@ -424,12 +422,77 @@ When enabled the user's password hash will be modified if it does not match the
 
 
 ==== Webhooks
-image::core-concepts/tenant-configuration-webhooks-settings.png[Tenant Configuration - Webhooks,width=1200,role=top-cropped]
+image::core-concepts/tenant-configuration-webhooks-settings.png[Tenant Configuration - Webhooks,width=1200,role=shadowed]
 
 ===== Table Columns
 
 include::docs/v1/tech/events-webhooks/_tenant-webhooks-table.adoc[]
 
+==== Security
+
+image::core-concepts/tenant-configuration-security-acl-catpcha.png[Tenant Configuration - Security: ACL and Captcha,width=1200,role=bottom-cropped]
+
+===== Login API settings
+
+[.api]
+[field]#Require an API key# [optional]#Optional#::
+When enabled the Login API will require an API key. This is functionally equivalent to requiring client authentication during OAuth2.
+
+===== Access control lists settings
+
+include::docs/v1/tech/shared/_premium-edition-blurb.adoc[]
+
+[.api]
+[field]#Access control list# [optional]#Optional#::
+The IP access control list that will be applied to this tenant.
+
+===== Captcha settings
+
+include::docs/v1/tech/shared/_premium-edition-blurb.adoc[]
+
+[.api]
+[field]#Enabled# [optional]#Optional#::
+When enabled, Captcha is used to help increase security of a form submission on the FusionAuth themed pages.
+
+[field]#Method# [required]#Required#::
```

Comment:
Not sure how other pages does this - but this is only required when enabled is `true`.